SDWebImage.xcodeproj/project.pbxproj

```diff
@@ -923,10 +923,11 @@
 			};
 			buildConfigurationList = 53922D69148C55810056699D /* Build configuration list for PBXProject "SDWebImage" */;
 			compatibilityVersion = "Xcode 3.2";
-			developmentRegion = English;
+			developmentRegion = en;
 			hasScannedForEncodings = 0;
 			knownRegions = (
 				en,
+				Base,
```

Comment:
Xcode automatic fix behavior with `Base` and `en`

Reply:
Need this be changed ? Or keep ?

Reply:
Keep